config/initializers/app_settings.rb

```diff
@@ -0,0 +1,24 @@
+class AppSettings
+  def self.load
+    config_file = File.join(Rails.root, "config", "settings.yml")
+    
+    if File.exists?(config_file)
+      all_config = YAML.load(File.read(config_file)) 
+      config = {}
+      ["default", Rails.env].each do |group|
+        config.merge!(all_config[group]) unless all_config[group].blank?
+      end
+
+      config.keys.reject{|k| k=="<<"}.each do |key|
```

Comment:
Surrounding space missing for operator '=='.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.